xnat_tools/logging.py

```diff
@@ -0,0 +1,31 @@
+import coloredlogs
+import logging
+import sys
+
+
+def setup_logging(
+    logger, logfile: str = "", verbose: bool = False, very_verbose: bool = False
+):
+    """Setup basic logging
+
+    Args:
+      loglevel (int): minimum loglevel for emitting messages
+    """
+
+    loglevel = logging.INFO
+    if very_verbose:
+        loglevel = logging.DEBUG
+    logformat = "[%(asctime)s] %(levelname)s:%(name)s:%(message)s"
+    handlers = None
+    if logfile == "":
+        handlers = [logging.StreamHandler(sys.stdout)]
+    else:
+        handlers = [logging.FileHandler(logfile), logging.StreamHandler(sys.stdout)]
```

Comment:
Nit: Since `logging.StreamHandler(sys.stdout)` is always a part of `handlers` I'd set that up front and optionally add the logfile. Something like:

```
handlers = [logging.StreamHandler(sys.stdout)]
if logfile:
  handlers.append(logging.FileHandler(logfile))
```

Just a nitpick though, it's obviously fine this way too :-)